uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,29 @@
+#
+# Raw Air Data.
+#
+
+#
+# Heater State
+# 
+uint8 HEATER_UNAVAILABLE    = 0
+uint8 HEATER_WORKING        = 1
+uint8 HEATER_OVERCURRENT    = 2
+uint8 HEATER_OPENCIRCUIT    = 3
+uint8 heater_state
```

Comment:
Please rename `heater_state` to `flags` in order to facilitate future extensibility of the data type. We may want to add new flags in the future that will have nothing to do with the heater. The constants should be renamed accordingly, e.g. `FLAG_HEATER_WORKING`, etc.

Concerning the field `FLAG_HEATER_UNAVAILABLE`, can we please avoid negative logic here, it's slightly more confusing than the reverse. I suggest renaming into `FLAG_HEATER_AVAILABLE`.